projects/reservoir/era5/daily_avg.py

```diff
@@ -0,0 +1,115 @@
+import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions
+import numpy as np
+import xarray as xr
+import xarray_beam as xbeam
+import argparse
+
+"""
+This script is tailored to provide daily averages from ERA5 data for a
+specific subset of variables used for creating an ocean RC model. The
+chunking and machines used are tailored to the specific use case so adjustments
```

Comment:
I think it's fine to hard code stuff in this script since our use case is very specific, but could you add a few more specifics about the hard coded assumptions here at the top? e.g. regrids to 1 deg x 1 deg, variables saved, etc.